utilities/ride-model-util/src/test/java/com/adobe/ride/utilities/model/test/ModelObjectTest.java

```diff
@@ -89,7 +89,7 @@ public void testStringJsonSchema() throws ModelSearchException {
     // pull in model as String
     try {
       InputStream inputStream =
-          ClassLoader.class.getResourceAsStream("/schemas/TestService/simple.json");
+          ModelObject.class.getResourceAsStream("/schemas/TestService/simple.json");
```

Comment:
Don't change this one.  ClassLoader is correct here.

Reply:
I've replace it with the manner used in the ModelObject class to fetch the schema string, and it works. As I've commented below, the develop branch fails this test due to a null pointer in this exact line. What should we do?

Reply:
Hmm, strange.  Let me check this out.

Reply:
Any news on this?

Reply:
Sorry, got tied up with another pressing issue.  Will have a response a little later today.

Reply:
So, I've run verify on the current develop branch and everything passes.

This is the command I used: mvn clean verify -Dgpg.skip

Tests run: 14, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.763 sec

Can you show me the command you are using to run your tests?

Reply:
I'm using `mvn -f build/pom.xml test`. I've tried now with `mvn -f build/pom.xml -Dgpg.skip verify` as you've recommended, and the test still fails. I'm on the origin/develop branch, up to date, working dir clean. Maybe it has something to do with my java version? I'm using java 13 on arch linux.

Reply:
Found it: the tests pass with java 8. Apparently there is a breaking change to java 13. What should we do?

Reply:
I'm downloading java 10 to test whether it works.

Reply:
The test fails with java 10.

Reply:
So, apparently this is our issue:
https://stackoverflow.com/questions/47613602/java-9-0-classloadergetresourceasstream-nullpointerexception

Reply:
I'll revert this change and keep the java 8 compatible version. The java 9+ compatibility issue can be addressed in another thread.

Reply:
Done, I believe this topic is settled now.